spec/sample-data/valid/includes/person-05.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "John Smith",
+  "company": {
+    "name": "Foo Incorporated",
+    "company_number": "12345"
+  },
+  "identifiers": [
+    {
+      "uid": "202918",
+      "identifier_system_code": "ab_cd"
+    }
+  ]
+}
```

Comment:
Can't quite tell why this file was added. Is it anything to do with `nationality`?